### ISSUE TITLE
Client startup failure (JS-related)

### ISSUE THREAD

**Issue by dsalt** [ValveSoftware/steam-for-linux#9321]
Devuan, mix of stable & testing; libc6 from testing.

This output is from Steam client build 1680578442, **launched from a terminal window**. If I start with `-no-cef-sandbox` then ~~I get #9320 instead~~ Steam starts up properly.
```
$ steam
steam.sh[3595]: Running Steam on devuan 4 64-bit
steam.sh[3595]: STEAM_RUNTIME is enabled automatically
setup.sh[3730]: Steam runtime environment up-to-date!
steam.sh[3595]: Steam client's requirements are satisfied
[2023-04-04 18:43:04] Startup - updater built Apr  4 2023 02:46:54
[2023-04-04 18:43:04] Startup - Steam Client launched with: '/home/steam/Steam/ubuntu12_32/steam'
[2023-04-04 18:43:04] Opted in to client beta 'publicbeta' via beta file
You are in the 'publicbeta' client beta.
04/04 18:43:04 Init: Installing breakpad exception handler for appid(steam)/version(1680578442)/tid(3846)
Looks like steam didn't shutdown cleanly, scheduling immediate update check
[2023-04-04 18:43:05] Loading cached metrics from disk (/home/steam/Steam/package/steam_client_metrics.bin)
[2023-04-04 18:43:05] Using the following download hosts for Public, Realm steamglobal
[2023-04-04 18:43:05] 1. https://client-update.akamai.steamstatic.com, /, Realm 'steamglobal', weight was 1000, source = 'update_hosts_cached.vdf'
[2023-04-04 18:43:05] 2. https://cdn.cloudflare.steamstatic.com, /client/, Realm 'steamglobal', weight was 1, source = 'update_hosts_cached.vdf'
[2023-04-04 18:43:05] 3. http://media.steampowered.com, /client/, Realm 'steamglobal', weight was 1, source = 'baked in'
[2023-04-04 18:43:05] Checking for update on startup
[2023-04-04 18:43:05] Checking for available updates...
[2023-04-04 18:43:05] Downloading manifest: https://client-update.akamai.steamstatic.com/steam_client_publicbeta_ubuntu12
[2023-04-04 18:43:05] Manifest download: send request
[2023-04-04 18:43:05] Manifest download: waiting for download to finish
[2023-04-04 18:43:05] Manifest download: finished
[2023-04-04 18:43:05] Download skipped: /steam_client_publicbeta_ubuntu12 version 1680578442, installed version 1680578442, existing pending version 0
[2023-04-04 18:43:05] Nothing to do
[2023-04-04 18:43:05] Verifying installation...
[2023-04-04 18:43:05] Performing checksum verification of executable files
[2023-04-04 18:43:05] Verification complete
ioctl (GFEATURE): Broken pipe
ioctl (GFEATURE): Broken pipe
ioctl (GFEATURE): Broken pipe
ioctl (GFEATURE): Broken pipe
ioctl (GFEATURE): Broken pipe
ioctl (GFEATURE): Broken pipe
Loaded SDL version 3.0.0-1396-gb6ae281e9
XRRGetOutputInfo Workaround: initialized with override: 0 real: 0xe7df6590
XRRGetCrtcInfo Workaround: initialized with override: 0 real: 0xe7df4dc0
GetWin32Stats: display was not open yet, good
ComputeStartupMode: found registry default startup mode: 0
steamwebhelper.sh[3865]: Runtime for steamwebhelper: defaulting to /home/steam/Steam/ubuntu12_64/steam-runtime-heavy
steamwebhelper.sh[3865]: glibc >= 2.34, partially disabling sandbox until CEF supports clone3()
Warning: failed to set thread priority: set failed for 8: -1: setpriority() failed
Warning: failed to set thread priority: set failed for priority 8
Warning: support for elevated priorities is most likely unavailable, suppressing future warnings
Warning: failed to set thread priority: set failed for 8: -1: setpriority() failed
PulseAudio connect failed (used only for Mic Volume Control) with error: Access denied
PulseAudio context init failed
CAppInfoCacheReadFromDiskThread took 138 milliseconds to initialize
Warning: failed to set thread priority: set failed for 8: -1: setpriority() failed
steamwebhelper.sh[3989]: Runtime for steamwebhelper: defaulting to /home/steam/Steam/ubuntu12_64/steam-runtime-heavy
steamwebhelper.sh[3989]: glibc >= 2.34, partially disabling sandbox until CEF supports clone3()
steamwebhelper.sh[4057]: Runtime for steamwebhelper: defaulting to /home/steam/Steam/ubuntu12_64/steam-runtime-heavy
steamwebhelper.sh[4057]: glibc >= 2.34, partially disabling sandbox until CEF supports clone3()
src/steamUI/steamuisharedjscontroller.cpp (352) : Failed creating offscreen shared JS context
src/steamUI/steamuisharedjscontroller.cpp (352) : Failed creating offscreen shared JS context
src/steamUI/steamuisharedjscontroller.cpp (352) : Fatal assert; application exiting
src/steamUI/steamuisharedjscontroller.cpp (352) : Fatal assert; application exiting
04/04 18:43:27 Init: Installing breakpad exception handler for appid(steam)/version(1680578442)/tid(3846)
```
(The current stable build requires that I use `-no-cef-sandbox` or I get no library view etc.)

[Edits: add terminal window note; the other startup-crash bug which this masked seems fixed.]

**Comment by MCPO-Spartan-117** [comment]
I can confirm that i'm also having this issue on a bedrock system with artix and gentoo, with steam coming from artix, however `-no-cef-sandbox` allows steam to start just fine.

[no-cef-steam.log](https://github.com/ValveSoftware/steam-for-linux/files/11272948/no-cef-steam.log)

[cef-steam.log](https://github.com/ValveSoftware/steam-for-linux/files/11272944/cef-steam.log)

**Comment by Seegras** [comment]
Tried the beta again today on Debian Sid, same error as dsalt reports. No matter whether run it with "-no-cef-sandbox" or not.

**Comment by DAMO238** [comment]
I can confirm this issue is also found an Arch. I have attached some relevant logs generated by steam.

[cef_log.txt](https://github.com/ValveSoftware/steam-for-linux/files/11524509/cef_log.txt)
[steamwebhelper.log](https://github.com/ValveSoftware/steam-for-linux/files/11524510/steamwebhelper.log)
[webhelper.txt](https://github.com/ValveSoftware/steam-for-linux/files/11524511/webhelper.txt)

**Comment by DAMO238** [comment]
I have found a workaround. The issue seems to be in the public beta, so the solution is to opt out. However, we cannot access the UI to do so, so we need to use the -clearbeta command line option. An issue arises when doing so though, were steam seems to be unable to connect to the internet despite having a solid connection. In order to resolve this, use --reset option immediately before opting out.

TL;DR, workaround is to run this:
```
steam --reset
steam -clearbeta
```

**Comment by dsalt** [comment]
@DAMO238, I think that you're reporting a different problem.

I see that you appear to be launching Steam with `-no-cef-sandbox` (as evidenced by `--no-sandbox` in the steamwebhelper launch commands).

Could you try launching _without_ that option, and from a terminal window? This is to confirm that Steam fails differently (and as described in this bug report) in that case.

**Comment by DAMO238** [comment]
I launched from a terminal with and without the -no-cef-sandbox, both of which had the same output, which seems to be the same issue that @Seegras  is having but slightly different to what you and @MCPO-Spartan-117 are having. (Please correct me if I am wrong in saying that.)

**Comment by RandomErrorMessage** [comment]
Same issue, rolling back from beta fixed. It didn't matter if I used -no-cef-sandbox or not.

**Comment by GregTheMadMonk** [comment]
Same issue here, Arch Linux 6.3.2 (Zen) + Nvidia

Here's the dump (gh won't allow `.dmp`, so I zipped it): [assert_20230523094130_28.dmp.zip](https://github.com/ValveSoftware/steam-for-linux/files/11540367/assert_20230523094130_28.dmp.zip)

And a log:
```
steam.sh[153934]: Running Steam on arch rolling 64-bit
steam.sh[153934]: STEAM_RUNTIME is enabled automatically
setup.sh[154060]: Steam runtime environment up-to-date!
steam.sh[153934]: Steam client's requirements are satisfied
[2023-05-23 09:40:40] Startup - updater built May 19 2023 22:00:10
[2023-05-23 09:40:40] Startup - Steam Client launched with: '/home/greg/.local/share/Steam/ubuntu12_32/steam'
[2023-05-23 09:40:40] Opted in to client beta 'publicbeta' via beta file
You are in the 'publicbeta' client beta.
05/23 09:40:40 Init: Installing breakpad exception handler for appid(steam)/version(1684535786)/tid(154577)
Looks like steam didn't shutdown cleanly, scheduling immediate update check
[2023-05-23 09:40:40] Loading cached metrics from disk (/home/greg/.local/share/Steam/package/steam_client_metrics.bin)
[2023-05-23 09:40:40] Using the following download hosts for Public, Realm steamglobal
[2023-05-23 09:40:40] 1. https://client-update.akamai.steamstatic.com, /, Realm 'steamglobal', weight was 1000, source = 'update_hosts_cached.vdf'
[2023-05-23 09:40:40] 2. https://cdn.cloudflare.steamstatic.com, /client/, Realm 'steamglobal', weight was 1, source = 'update_hosts_cached.vdf'
[2023-05-23 09:40:40] 3. http://media.steampowered.com, /client/, Realm 'steamglobal', weight was 1, source = 'baked in'
[2023-05-23 09:40:40] Checking for update on startup
[2023-05-23 09:40:40] Checking for available updates...
[2023-05-23 09:40:40] Downloading manifest: https://client-update.akamai.steamstatic.com/steam_client_publicbeta_ubuntu12
[2023-05-23 09:40:40] Manifest download: send request
[2023-05-23 09:40:40] Manifest download: waiting for download to finish
[2023-05-23 09:40:40] Manifest download: finished
[2023-05-23 09:40:40] Download skipped: /steam_client_publicbeta_ubuntu12 version 1684535786, installed version 1684535786, existing pending version 0
[2023-05-23 09:40:40] Nothing to do
[2023-05-23 09:40:40] Verifying installation...
[2023-05-23 09:40:40] Performing checksum verification of executable files
[2023-05-23 09:40:41] Verification complete

(process:154577): GLib-GObject-CRITICAL **: 09:40:41.958: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

(process:154577): GLib-GObject-CRITICAL **: 09:40:41.958: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
Loaded SDL version 3.0.0-1540-gd5f07730b

(steam:154577): Gtk-WARNING **: 09:40:42.014: Unable to locate theme engine in module_path: "adwaita",
/usr/share/themes/amarena/gtk-2.0/main.rc:805: error: unexpected identifier 'direction', expected character '}'

(steam:154577): Gtk-WARNING **: 09:40:42.017: Unable to locate theme engine in module_path: "adwaita",
/usr/share/themes/amarena/gtk-2.0/hacks.rc:28: error: invalid string constant "normal_entry", expected valid string constant
XRRGetOutputInfo Workaround: initialized with override: 0 real: 0xf2e69db0
XRRGetCrtcInfo Workaround: initialized with override: 0 real: 0xf2e68500
GetWin32Stats: display was not open yet, good
ComputeStartupMode: found registry default startup mode: 0
Switching to desktopui, since -vgui was not specified
steamwebhelper.sh[160594]: Runtime for steamwebhelper: defaulting to /home/greg/.local/share/Steam/ubuntu12_64/steam-runtime-heavy
steamwebhelper.sh[160594]: glibc >= 2.34, partially disabling sandbox until CEF supports clone3()
CAppInfoCacheReadFromDiskThread took 66 milliseconds to initialize
src/steamUI/steamuisharedjscontroller.cpp (448) : Failed creating offscreen shared JS context
src/steamUI/steamuisharedjscontroller.cpp (448) : Failed creating offscreen shared JS context
src/steamUI/steamuisharedjscontroller.cpp (448) : Fatal assert; application exiting
src/steamUI/steamuisharedjscontroller.cpp (448) : Fatal assert; application exiting
05/23 09:41:30 Init: Installing breakpad exception handler for appid(steam)/version(1684535786)/tid(154577)
assert_20230523094130_28.dmp[165907]: Uploading dump (out-of-process)
/tmp/dumps/assert_20230523094130_28.dmp
_ExitOnFatalAssert

assert_20230523094130_28.dmp[165907]: Finished uploading minidump (out-of-process): success = yes
assert_20230523094130_28.dmp[165907]: response: CrashID=bp-0d350791-4fed-46b1-839d-eccee2230522
```

**Comment by GregTheMadMonk** [comment]
Any progress on the issue?

**Comment by SteelDrake** [comment]
The UI overhaul beta seems to work but the usual one seems to still refuse to start properly. `-no-cef-sandbox` doesn't do anything for me. I'm on endeavourOS.

**Comment by GregTheMadMonk** [comment]
It it the most weird thing: on Arch, after updating and deleting all `xdg-desktop-portal-*` packages except `xdg-desktop-portal-gtk` steam works with both betas just fine. I'm starting to thing that on Arch it was not a Steam problem as I've started experiencing problems with all GTK apps around the same time, and all the problems disappeared with this solution

**Comment by kisak-valve** [comment]
Hello @GregTheMadMonk, are you able to check your package manager logs and list what packages you removed and what versions they were? It would be useful to isolate the change in behavior to the presence of specific system package(s).

**Comment by GregTheMadMonk** [comment]
[pacman.log.fragment.txt](https://github.com/ValveSoftware/steam-for-linux/files/11645803/pacman.log.fragment.txt)
Here's the ending of my `pacman.log` corresponding to June 4th. There's quite a lot, but I'm relatively sure only the last several lines are relevant

**Comment by kisak-valve** [comment]
For search-ability, I'll repeat the key details of that log. 
```
[ALPM] removed xdg-desktop-portal-gnome (44.1-2)
[...]
[ALPM] removed xdg-desktop-portal-hyprland (0.4.0-1)
```

Now the question is if one or both of those are common to all the affected systems, and if it's a system package regression not seen with an older version of the package.

**Comment by GregTheMadMonk** [comment]
I'll try to reinstall the packages to check if they actually cause things to break

**Comment by SteelDrake** [comment]
I looked at my installed packages and noticed I had `xdg-desktop-portal-gnome`. After removing it and redownloading the beta, it worked just fine. I even checked by restarting Steam, started just like it should.

**Comment by dsalt** [comment]
That xdg-desktop-portal-* problem looks very much like a different problem: their presence or absence make no difference here. (I have Steam in a chrooted environment.)

**Comment by Seegras** [comment]
1686092969 works again for me, expect everything is tiny, but that's another issue.

**Comment by dsalt** [comment]
> 1686092969 works again for me, expect everything is tiny, but that's another issue.
I've just re-tested with that build. Still fine with `-no-cef-sandbox`, still fails with that JS context error without.

**Comment by n3oney** [comment]
I get this problem on latest NixOS. Can't even log in, gray screen. Same error, `src/steamUI/steamuisharedjscontroller.cpp (421) : Failed creating offscreen shared JS context`

**Comment by billksun** [comment]
Same error here as @n3oney on NixOS. Steam version 1685488080.

**Comment by samueldr** [comment]
Hi, noting here that the NixOS issue at this moment, while exhibiting the same symptoms superficially, is 99% unlikely to be related to the other issues reported here. The NixOS issue [is caused by our specific peculiarities, and a fix is merged](https://github.com/NixOS/nixpkgs/pull/236606).

**Comment by the-science-guy** [comment]
> Same error here as @n3oney on NixOS. Steam version 1685488080.
> 
> Edit: `steam --reset` and `steam -clearbeta` didn't resolve the issue. Removing the `xdg-desktop-portal-*` package doesn't seem to do anything either.

Not only was steam affected, but other applications as well (librewolf took ages to start for example). The culprit was `xdg-desktop-portal-gnome` being installed in addition to
```xdg-desktop-portal xdg-desktop-portal-gtk xdg-desktop-portal-xapp```
as the latter three were all dependencies of `cinnamon` and could not be removed. I suspect some interaction between the gtk and gnome packages interfering with each other.

Removing  `xdg-desktop-portal-gnome`  not only fixed steam, but returned other application behavior to its original state.

**Comment by dsalt** [comment]
> Removing `xdg-desktop-portal-gnome` not only fixed steam, but returned other application behavior to its original state.

Okay… if that's the case then why am I seeing that JS context failure (without `-no-cef-sandbox`), given that that xdg package has never been installed here?

**Comment by dozn** [comment]
I can confirm that removing `xdg-desktop-portal-gnome` resolved this issue for me on Arch.

Worth noting for anyone else still having issues, I was successfully running steam 9/10 times by using `DEBUGGER=gdb steam`, hitting `y` when asked "Enable debuginfod for this session?" (won't work if you hit `n` instead), then entering `run`.

**Comment by dsalt** [comment]
@lostgoat, all that I see is this:
```
[0615/204427.248264:INFO:crash_reporting.cc(248)] Crash reporting enabled for process: browser
[0615/204427.249088:FATAL:zygote_host_impl_linux.cc(117)] No usable sandbox! Update your kernel or see https://chromium.googlesource.com/chromium/src/+/master/docs/linux/suid_sandbox_development.md for more information on developing with the SUID sandbox. If you want to live dangerously and need an immediate workaround, you can try using --no-sandbox.
```
```
[1970-01-01 01:00:00] Client version: no bootstrapper found
[1970-01-01 01:00:00] Startup - webhelper launched with: /home/steam/Steam/ubuntu12_64/steamwebhelper --disable-seccomp-filter-sandbox -lang=en_US -cachedir=/home/steam/Steam/config/htmlcache -steampid=32020 -buildid=1686779606 -steamid=0 -logdir=/home/steam/Steam/logs -uimode=7 -startcount=2 -steamuniverse=Public -realm=Global -clientui=/home/steam/Steam/clientui -steampath=/home/steam/Steam/ubuntu12_32/steam --enable-media-stream --disable-smooth-scrolling --disable-accelerated-video-decode --password-store=basic --log-file=/home/steam/Steam/logs/cef_log.txt --disable-quick-menu --disable-features=SameSiteByDefaultCookies --enable-blink-features=ResizeObserver,Worklet,AudioWorklet --disable-blink-features=Badging
```
The referenced page gives no obvious clues.

On the kernel side:
```
CONFIG_CGROUPS=y
# CONFIG_CGROUP_FAVOR_DYNMODS is not set
CONFIG_BLK_CGROUP=y
CONFIG_CGROUP_WRITEBACK=y
CONFIG_CGROUP_SCHED=y
CONFIG_CGROUP_PIDS=y
CONFIG_CGROUP_RDMA=y
CONFIG_CGROUP_FREEZER=y
CONFIG_CGROUP_DEVICE=y
CONFIG_CGROUP_CPUACCT=y
CONFIG_CGROUP_PERF=y
CONFIG_CGROUP_MISC=y
CONFIG_CGROUP_DEBUG=y
# CONFIG_BLK_CGROUP_IOLATENCY is not set
# CONFIG_BLK_CGROUP_IOCOST is not set
# CONFIG_BLK_CGROUP_IOPRIO is not set
# CONFIG_NETFILTER_XT_MATCH_CGROUP is not set
# CONFIG_CGROUP_NET_PRIO is not set
# CONFIG_CGROUP_NET_CLASSID is not set
```
```
CONFIG_HAVE_ARCH_SECCOMP=y
CONFIG_HAVE_ARCH_SECCOMP_FILTER=y
CONFIG_SECCOMP=y
CONFIG_SECCOMP_FILTER=y
# CONFIG_SECCOMP_CACHE_DEBUG is not set
```

**Comment by kohotli** [comment]
I'm having this issue (seeing the shared JS context message in a terminal) and running with -no-cef-sandbox doesn't change anything for me. Removing xdg-desktop-portal-gnome didn't make a difference for me. I'm on Arch, not running the client beta.

**Comment by starcraft66** [comment]
Having the same shared JS context error:

```
@luna:~ ❯ cat ~/.steam/steam/logs/cef_log.txt                                            
[0616/110459.586804:INFO:crash_reporting.cc(248)] Crash reporting enabled for process: browser
[0616/110459.630205:INFO:crash_reporting.cc(271)] Crash reporting enabled for process: gpu-process
```

**Comment by n3f4s** [comment]
I have the same issue. `steam --reset; steam -clearbeta` works but needing to reset every times I want to start steam is annoying (and it reset the shared libraries).

I haven't tried removing any `xdg-desktop-portal*` as they are required for software that I use regularly.

`steam -no-cef-sandbox` behave exactly as without this option (there's a crash).

**Comment by diinki** [comment]
Just throwing in that I have the same issue, on Arch _btw_. I noticed that it happens a lot more after the new UI update.

```
src/steamUI/steamuisharedjscontroller.cpp (529) : Failed creating offscreen shared JS context
src/steamUI/steamuisharedjscontroller.cpp (529) : Failed creating offscreen shared JS context
src/steamUI/steamuisharedjscontroller.cpp (529) : Fatal assert; application exiting
src/steamUI/steamuisharedjscontroller.cpp (529) : Fatal assert; application exiting
```

**Comment by LordHorst** [comment]
Had the same problem, and I am on Arch as well, removing `xdg-desktop-portal-gnome` did indeed fix the problem for me. `steam --reset` and `steam -clearbeta` unfortunatly didn't. What's most surprising for me is that steam worked fine until the day before yesterday, and I didn't notice any updates or upgrades, not on steam and not on my system. But I am a "fire and forget" kind of guy when it comes to updates. I see the notification, run the update, and promptly forget about it. So that's not saying much...

**Comment by shoemark** [comment]
I seemed to have the same problem on Gentoo Linux for the past couple of days, except in about 3% of the attempts to start Steam, it would actually start. The `--reset`, `-no-cef-sandbox` and `-clearbeta` flags did not seem to affect this chance. Removing `xdg-desktop-portal-gnome`, however, did indeed fix the problem for me.

**Comment by alfonsodg** [comment]
> xdg-desktop-portal-gnome

Remove this package also works in Manjaro and also improves radically the load of some applications like libreoffice calc, telegram desktop, etc

**Comment by starcraft66** [comment]
I had `xdg-desktop-portal-gnome` enabled on my machine and leftover from some testing. I don't actually even use it for anything. I disabled it per suggestions here and in the upstream thread, now steam opens and works fine...

**Comment by n3f4s** [comment]
> I have the same issue. `steam --reset; steam -clearbeta` works but needing to reset every times I want to start steam is annoying (and it reset the shared libraries).
> 
> I haven't tried removing any `xdg-desktop-portal*` as they are required for software that I use regularly.
> 
> `steam -no-cef-sandbox` behave exactly as without this option (there's a crash).

Update: removing the `xdg-desktop-portal-gnome` package doesn't solve the issue

```
vincent@shodan ~> pacman -Qs xdg-desktop-portal-gnome
vincent@shodan ~ [1]> steam
steam.sh[2238]: Running Steam on arch rolling 64-bit
steam.sh[2238]: STEAM_RUNTIME is enabled automatically
...
src/steamUI/steamuisharedjscontroller.cpp (529) : Failed creating offscreen shared JS context
src/steamUI/steamuisharedjscontroller.cpp (529) : Failed creating offscreen shared JS context
src/steamUI/steamuisharedjscontroller.cpp (529) : Fatal assert; application exiting
src/steamUI/steamuisharedjscontroller.cpp (529) : Fatal assert; application exiting
06/18 14:19:20 Init: Installing breakpad exception handler for appid(steam)/version(1686880776)/tid(2367)
assert_20230618141920_30.dmp[2570]: Uploading dump (out-of-process)
/tmp/dumps/assert_20230618141920_30.dmp
_ExitOnFatalAssert

**Comment by kohotli** [comment]
Update to my post I made earlier, removing xdg-desktop-portal-gnome didn't fix the problem, but removing xdg-desktop-portal and xdg-desktop-portal-gtk did

**Comment by Alfagun74** [comment]
Same error on WSL, neither ``steam --reset; steam -clearbeta`` nor removing ``xdg-desktop-portal-*`` helped me.

**Comment by D1Ck3n** [comment]
`sudo pacman -R xdg-desktop-portal-gnome` fixed it for me

**Comment by lostgoat** [comment]
@dsalt the latest client should detect that the chrome sandbox is unsupported and fall back gracefully to no sandbox mode.

Let me know if you still need to specify the no sandbox cli flag on your system.

**Comment by dsalt** [comment]
@lostgoat, 1687306661 failing as before; 1687386907 starts up. There is an indication of partial disablement related to clone3(), but I saw no further messages concerning sandboxing.

More diagnostic output would be good; if I can find what's stopping the sandboxing from working…

**Comment by vv4r10ck** [comment]
removing xdg-desktop-portal-* did it for me!

I also had to uninstall flatpak and lutris

**Comment by morysol** [comment]
I just remove xdg-desktop-portal-gnome and steam works fine.
lutris was not touched. No problem with my games.
Manjaro, Nvidia 1070
thanx for advice, btw ;)

**Comment by rykugur** [comment]
Can confirm that deleting xdg-desktop* packages (which also meant I had to delete cinnamon, which I don't use) fixed the issue for me - I was able to launch the steam desktop app normally.

Running endeavouros

**Comment by sparr** [comment]
I started encountering this failure today after performing an update to my Manjaro (Arch-based) system. Full client output follows at https://gist.github.com/sparr/b1586c7bcb3ea4f85a62ba60b645dea8

**Comment by lostgoat** [comment]
> More diagnostic output (or info about what to test!) would be good; if I can find what's stopping the sandboxing from working…

@dsalt we probably won't add extra logging here since that is part of the chrome stack (and we prefer to avoid chrome patches). But since the code is open source I can link you to all the checks that are happening, hope that is helpful:
https://chromium.googlesource.com/chromium/src/+/lkgr/sandbox/linux/services/credentials.cc#264

Note, you should also see the following message in your webhelper.txt logfile:
```
Disabling sandbox due to a previous crash in CefInitialize with the sandbox enabled
```

**Comment by dsalt** [comment]
Something's presumably changed locally (but what? All that I can think of is a recent restart of X) as I'm not seeing that message re. CefInitialize – looks to me like all is well.

**Comment by ZakNilsen** [comment]
![image](https://github.com/ValveSoftware/steam-for-linux/assets/86334244/033119d6-cbb2-46ef-a6dc-3844f69abdc8)

Having issues with Ubuntu 22.04, just installed steam and am getting these errors when I try to launch it

**Comment by sigboe** [comment]
uninstalling xdg-desktop-portals* and all programs depending on it worked for me on Arch Linux

**Comment by ds0nt** [comment]
> xdg-desktop-portal-*

Amazing. Worked for me on arch, thanks for this.

**Comment by AnthonyMichaelTDM** [comment]
On arch linux, removing xdg-desktop-portal-gnome fixed the issue for me

**Comment by Alfagun74** [comment]
Anyone getting the "new UI" working on WSL?

**Comment by ydm** [comment]
> I launched from a terminal with and without the -no-cef-sandbox, both of which had the same output, which seems to be the same issue that @Seegras is having but slightly different to what you and @MCPO-Spartan-117 are having. (Please correct me if I am wrong in saying that.)

Big thanks! Deleted `xdg-desktop-portal*` and it works fine.

**Comment by kisak-valve** [comment]
At this point the comments that are trickling into here are intended for #9588.

The symptom in the title is generic and there might be other issues in the future that look similar and need to be pondered separately, but that's not related to the issue tracked in the opening post of this issue report.
